services/libssh/config.json

```diff
@@ -0,0 +1,34 @@
+{
+    "event_types": [
+        {
+            "name": "cve_2018_10933_ssh_auth_bypass",
+            "label": "CVE-2018-10933 SSH Bypass",
+            "fields": ["originating_ip", "originating_port"],
+            "policy": "Alert"
+        },
+        {
+            "name": "ssh",
+            "label": "SSH Interaction",
+            "fields": ["originating_ip", "originating_port", "username", "password", "additional_fields"],
+            "policy": "Alert"
+        }
+    ],
+    "service": {
```

Comment:
Add a description for this honeypot and specify the detected vulnerability there. As an example look at the description for the micros honeypot.